CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.8)
 
+set(WICKED_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR} CACHE INTERNAL "")
```

Comment:
I don't think this needs to be a CACHE entry, as we don't want to expose this variable to the user configuring the project

Reply:
You're right. i originally did that to make it acceasable from parent directories, but realise now that's silly, since the path should be known anyway. I'll fix that today or tomorrow 😃